gvsbuild/projects/libfido2.py

```diff
@@ -0,0 +1,27 @@
+# from gvsbuild.utils.base_expanders import GitRepo
+from gvsbuild.utils.base_project import Project, project_add
+from gvsbuild.utils.base_expanders import GitRepo
+import os
```

Comment:
cosmetic: other files import os before the gvsbuild local modules. Also include the license blurb for consistency